lib/postrank-uri.rb

```diff
@@ -211,7 +211,7 @@ def c14n(uri, opts = {})
     def embedded(uri)
       embedded = if uri.host == 'news.google.com' && uri.path == '/news/url' \
          || uri.host == 'xfruits.com'
-        uri.query_values['url']
+        uri.query_values.try!(:[], 'url')
```

Comment:
This gem is not a rails/activesupport project so `try!` isn't available, and I don't think we should add activesupport as a dependency because we don't want to get too far away from upstream.

```
  1) PostRank::URI canonicalization embedded links should extract embedded redirects from Google News
     Failure/Error: PostRank::URI.c14n(uri).to_s
     NoMethodError:
       undefined method `try!' for #<Hash:0x007fe72d84b040>
     # ./lib/postrank-uri.rb:214:in `embedded'
     # ./lib/postrank-uri.rb:191:in `c14n'
     # ./spec/postrank-uri_spec.rb:177:in `c'
     # ./spec/postrank-uri_spec.rb:229:in `block (4 levels) in <top (required)>'
```

Should probably add a spec for this case also.